save-frontend/src/main/kotlin/org/cqfn/save/frontend/utils/RequestUtils.kt

```diff
@@ -52,7 +54,7 @@ suspend fun get(url: String, headers: Headers) = request(url, "GET", headers)
  * @param body request body
  * @return [Response] instance
  */
-suspend fun post(url: String, headers: Headers, body: dynamic) = request(url, "POST", headers, body)
+suspend fun post(url: String, headers: Headers, body: dynamic) = request(url, "POST", headers, body, RequestCredentials.SAME_ORIGIN)
```

Comment:
and what's with get request?

Reply:
That's actually the default value of this parameter, and it's not required to set it explicitly. I guess I left it here after experimenting. So, `get` implicitly has the same value, I'll remove it from here too.